integration-tests/tests/test_router.py

```diff
@@ -0,0 +1,121 @@
+import pytest
+import time
+import uuid
+from conftest import user_1, user_2, option
+from test_cases import SignalTestCase
+
+
+@pytest.mark.rpc
+@pytest.mark.transaction
+@pytest.mark.wallet
+class TestTransactionFromRoute(SignalTestCase):
+
+    await_signals = [
+        "wallet.suggested.routes",
+        "wallet.router.sign-transactions",
+        "wallet.router.sending-transactions-started",
+        "wallet.transaction.status-changed",
+        "wallet.router.transactions-sent"
+    ]
+
+    def test_tx_from_route(self):
+
+        _uuid = str(uuid.uuid4())
+        amount_in = "0xde0b6b3a7640000"
+
+        method = "wallet_getSuggestedRoutesAsync"
+        params = [
+            {
+                "uuid": _uuid,
+                "sendType": 0,
+                "addrFrom": user_1.address,
+                "addrTo": user_2.address,
+                "amountIn": amount_in,
+                "amountOut": "0x0",
+                "tokenID": "ETH",
+                "tokenIDIsOwnerToken": False,
+                "toTokenID": "",
+                "disabledFromChainIDs": [10, 42161],
```

Comment:
@antdanchenko since we're resolving selected chains based on the entry in the `networks` table, and since the statusgo is run with the config set in `integration-tests/config.json` (that provides config for the Anvil chain only), the result will be the same either we provide an empty list here or the list with all but the Anvil chain id.

Reply:
thanks for the note, we will add more anvil networks and bridges in future to simulate proper route, but for now let's keep as it is